bundles/automation/org.eclipse.smarthome.automation.module.core/META-INF/MANIFEST.MF

```diff
@@ -5,6 +5,7 @@ Bundle-SymbolicName: org.eclipse.smarthome.automation.module.core
 Bundle-Version: 0.9.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Import-Package: com.google.common.collect,
+ org.apache.commons.lang,
```

Comment:
Also remember the discussion about Commons Lang3, so whereever possible, I would refrain from using the lib at places where we do not yet have any dependency on it.